packages/services/src/services/Translation/Translation.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
+import root from 'window-or-global';
+
+/**
+ * @constant {string | string} Host for the Translation API call
+ * @private
+ */
+const _host = process.env.TRANSLATION_HOST || 'https://www.ibm.com';
+
+/**
+ * @constant {string | string} CORS proxy for lower environment calls
+ * @private
+ */
+const _proxy = process.env.CORS_PROXY || '';
+
+/**
+ * Translation API endpoint
+ *
+ * @type {string}
+ * @private
+ */
+const _endpoint = `${_host}/common/v18/js/data/jsononly`;
+
+/**
+ * Translation API class with methods for fetching i18n data for ibm.com
+ */
+class TranslationAPI {
+  /**
+   * Returns translation i18n data
+   *
+   * @returns {Promise<any>} Translation data
+   * @example
+   * import { TranslationAPI } from '@carbon/ibmdotcom-services';
+   *
+   * async function getTranslation() {
+   *   const response = await TranslationAPI.getTranslation();
+   *   return response;
+   * }
+   */
+  static async getTranslation() {
```

Comment:
I wonder if it'd be good to provide may be at least two additional methods. `.getTranslationMasthead()`, and `.getTranslationFooter()`. That might give us flexibility down the road to scale, and depending on how Drupal delivers it'd allow us to split the two calls up if needed/desired.